pages/labTwo.html

```diff
@@ -1,39 +1,170 @@
-<!DOCTYPE html>
-<html lang="en">
+<!DOCTYPE HTML>
+<html lang="en-US">
 <head>
     <meta charset="UTF-8">
-    <title>Lab Project 2</title>
-    <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Droid+Serif|Lobster" />
-    <style>
+    <title>Patient Information Site</title>
+    <style type="text/css">
         body {
-            background-color: #ffffcc;
+            margin: 2em 5em;
+            font-family: Georgia, "Times New Roman", Times, serif;
         }
 
-        body>p {
-            color: #006600;
-            margin-left: 20px;
-            font-size: 65px;
-            text-align: center;
-            font-family: "Lobster", cursive, serif;
+        h1, legend {
+            font-family: Arial, Helvetica, sans-serif;
         }
-        .footer {
-            bottom : 2px;
-            height : 40px;
-            margin-top : 40px;
-            text-align: center;
-            vertical-align: middle;
-            position:fixed;
-            width:100%;
+
+        label, input, select {
+            display: block;
+        }
+
+        input, select {
+            margin-bottom: 1em;
+        }
+
+        fieldset {
+            margin-bottom: 2em;
+            padding: 1em;
+        }
+
+        fieldset fieldset {
+            margin-top: 1em;
+            margin-bottom: 1em;
         }
     </style>
 </head>
 <body>
-<p> Coming soon!! Keep in touch</p>
-<div class="footer">
+<form id="register">
+    <h1>Patient Registration Form</h1>
+    <fieldset>
+        <legend>Personal Details</legend>
+        <div title="Use this field for identifying the patient by name or his/her patient ID">
```

Comment:
I would recommend adding help text for clarifying the intent of each data collection point; adding a `title` like this will display the given text when the user hovers over the field. Additionally, it would be good to add a widget to click (like a question mark icon) to make it more explicit; the user may not realize they need to hover to pop the help text up.